src/otelcollector/otelcol-config.yml

```diff
@@ -16,14 +20,19 @@ exporters:
 
 processors:
   batch:
+  spanmetrics:
+    metrics_exporter: prometheus
 
 service:
   pipelines:
     traces:
       receivers: [otlp]
-      processors: [batch]
+      processors: [spanmetrics,batch]
       exporters: [logging, jaeger]
     metrics:
       receivers: [otlp]
       processors: [batch]
       exporters: [prometheus, logging]
+    metrics/spanmetrics:
+      receivers: [ otlp/spanmetrics ]
+      exporters: [ prometheus ]
```

Comment:
Can we add the logging exporter to this as well?

Reply:
done